client/lib/purchases/assembler.js

```diff
@@ -72,6 +72,7 @@ function createPurchaseObject( purchase ) {
 		siteId: Number( purchase.blog_id ),
 		siteName: purchase.blogname,
 		subscribedDate: purchase.subscribed_date,
+		subscribedMoment: purchase.subscribed_date ? i18n.moment( purchase.subscribed_date ) : null,
```

Comment:
Feels so weird to have a duplicated value here, but if we're doing that already for other props, let's follow suite.

Reply:
Yeah it's a bit odd, but also I prefer to not reinvent the wheel here :)